localization/react-intl/src/app/components/UserTosForm.json

```diff
@@ -5,7 +5,7 @@
     "defaultMessage": "Terms of Service"
   },
   {
-    "id": "userTos.ppLink",
+    "id": "userTos.ppLink\"",
```

Comment:
Is this a typo?

Reply:
Thanks for catching this @amoedoamorim it was a typo...the dreaded copy-paste kind of typo!